src/amplitude.js

```diff
@@ -13,83 +14,53 @@ var UUID = require('./uuid');
 var version = require('./version');
 var DEFAULT_OPTIONS = require('./options');
 
-var IDENTIFY_EVENT = '$identify';
-var API_VERSION = 2;
-var MAX_STRING_LENGTH = 1024;
-var LocalStorageKeys = {
-  LAST_EVENT_ID: 'amplitude_lastEventId',
-  LAST_EVENT_TIME: 'amplitude_lastEventTime',
-  LAST_IDENTIFY_ID: 'amplitude_lastIdentifyId',
-  LAST_SEQUENCE_NUMBER: 'amplitude_lastSequenceNumber',
-  REFERRER: 'amplitude_referrer',
-  SESSION_ID: 'amplitude_sessionId',
-
-  // Used in cookie as well
-  DEVICE_ID: 'amplitude_deviceId',
-  OPT_OUT: 'amplitude_optOut',
-  USER_ID: 'amplitude_userId'
-};
-
-/*
- * Amplitude API
+/**
+ * Amplitude SDK API - instance constructor.
+ * @constructor Amplitude
+ * @public
+ * @example var amplitude = new Amplitude();
  */
-var Amplitude = function() {
+var Amplitude = function Amplitude() {
   this._unsentEvents = [];
   this._unsentIdentifys = [];
   this._ua = new UAParser(navigator.userAgent).getResult();
   this.options = object.merge({}, DEFAULT_OPTIONS);
   this.cookieStorage = new cookieStorage().getStorage();
   this._q = []; // queue for proxied functions before script load
-};
+  this._sending = false;
+  this._updateScheduled = false;
 
-Amplitude.prototype._eventId = 0;
-Amplitude.prototype._identifyId = 0;
-Amplitude.prototype._sequenceNumber = 0;
-Amplitude.prototype._sending = false;
-Amplitude.prototype._lastEventTime = null;
-Amplitude.prototype._sessionId = null;
-Amplitude.prototype._newSession = false;
-Amplitude.prototype._updateScheduled = false;
+  // event meta data
```

Comment:
We were doing something weird with declaring instance variables in both the constructor and on the prototype. We should just stick everything in the constructor